doc/api/cli.md

```diff
@@ -1527,8 +1539,8 @@ Preload the specified module at startup.
 Follows `require()`'s module resolution
 rules. `module` may be either a path to a file, or a node module name.
 
-Only CommonJS modules are supported. Attempting to preload a
-ES6 Module using `--require` will fail with an error.
+Only CommonJS modules are supported.
+Use [`--import`][] to load a [ECMAScript modules][].
```

Comment:
```suggestion
Use [`--import`][] to preload an [ECMAScript module][].
```

Reply:
This is a grammatical error, it should be fixed before landing.

Reply:
will do